index.bs

```diff
@@ -728,14 +720,6 @@ consisting of the character U+00C5 LATIN CAPITAL LETTER A WITH RING ABOVE.</p>
   lt="WebVTT cue middle alignment">middle</a>, if there is no <a>WebVTT cue text alignment</a>
   setting for a cue, the <a>WebVTT cue position</a> defaults to 50%.</p>
 
-  <p class="note">Even for <a lt="WebVTT cue horizontal writing direction">horizontal</a> cues with
```

Comment:
I can put it back. Although I disagree with the advice in the last paragraph; see https://www.w3.org/Bugs/Public/show_bug.cgi?id=28257#c12